spec/mam.js

```diff
@@ -265,6 +265,60 @@ describe("Message Archive Management", function () {
                 done();
             }));
 
+            it("is not discarded if it comes from the right sender",
+                mock.initConverse(
```

Comment:
Can you please add a forged message to this test-case and then check that it gets rejected?

That way we cover both cases.

I also think the test can then be renamed to `is discarded if it's forged`, since that's the exception and being not discarded is the norm.

Reply:
The other case is already covered [here](https://github.com/conversejs/converse.js/blob/master/spec/mam.js#L213). That's why I didn't add it.